openapi.yaml

```diff
@@ -1341,6 +1346,16 @@ components:
       scheme: bearer
       bearerFormat: JWT
   schemas:
+    JwtRequest:
+      type: object
+      properties:
+        hoursUntilExpiry:
+          type: integer
```

Comment:
minimum is 1

Reply:
added approval, would change it here but i'm not sure you want me to, and i need access if i wanted to update swaggerdocs, lemme know/share with me if you want me to update this to 1 in both places